store/storetest/channel_store.go

```diff
@@ -5569,6 +5580,13 @@ func testChannelStoreSearchArchivedInTeam(t *testing.T, ss store.Store, s SqlSto
 		require.Error(t, err)
 		require.Nil(t, list)
 	})
+
+	t.Run("find term", func(t *testing.T) {
+		list, err := ss.Channel().SearchArchivedInTeam(teamId, "Channel", userId)
+		require.NoError(t, err)
+		require.NotNil(t, list)
+		require.NotEmpty(t, list)
```

Comment:
Can we check the result contains the expected channel?

Reply:
sure!